### ISSUE TITLE
Firebase API Key in Repo?

### ISSUE THREAD

**Issue by ecc521** [ecc521/SignMeIn#8]

**Comment by FloppyT** [comment]
No, should keep in a local config file

**Comment by ecc521** [comment]
@FloppyT, Yes, I'm aware of that, and do that for my personal projects. This was
mostly intended as a warning and reminder to @imharris-oss that a new key will have to be made, stored locally, and the old key terminated.

More extreme action did not seem to be required, given the non-use of the API key, and restrictions imposed on it.
